config/src/projects/habitat.ts

```diff
@@ -0,0 +1,79 @@
+import { Project } from './Project'
+import { tokenList } from '../tokens'
+
+export const habitat: Project = {
+  name: 'Habitat',
+  bridges: [
+    {
+      address: '0x96E471B5945373dE238963B4E032D3574be4d195',
+      sinceBlock: 12731872,
+      tokens: tokenList.map((e) => e.symbol),
```

Comment:
Please limit the tokens to the once the rollup actually holds. I this case those are: HBT, WETH, USDC and GNO. The more tokens there are the more data we need to process, so it is best to keep this list lean :)